site/content/docs/5.2/components/navbar.md

```diff
@@ -732,6 +732,52 @@ To create an offcanvas navbar that expands into a normal navbar at a specific br
 </nav>
 ```
 
+When using offcanvas in a dark navbar, be aware that you may need to have a dark background on the offcanvas content to avoid the text becoming illegible. In the example below, we add `.navbar-dark` and `.bg-dark` to the `.navbar`, `.text-bg-dark` to the `.offcanvas`, `.dropdown-menu-dark` to `.dropdown-menu`, and `.btn-close-white` to `.btn-close` for proper styling with a dark offcanvas.
```

Comment:
Proposition of enhancement (not tested the link format). I would have used a direct link to the Dark variant of the offcanvas rather than explaining how it is built.

```suggestion
When using offcanvas in a dark navbar, be aware that you may need to have a dark background on the offcanvas content to avoid the text becoming illegible. In the example below, we add `.navbar-dark` and `.bg-dark` to the `.navbar`, `.dropdown-menu-dark` to `.dropdown-menu`, and use a [Dark offcanvas]({{< docsref "/components/offcanvas/#dark-offcanvas" >}})`.
```